Debugging Lesson 5.1

VadimDmitr opened the pull request:
Debugging Lesson 5.1

VadimDmitr committed:
Debugging Lesson 5.1

kiramiller425 commented:
Hi Vadim,
I looked at your lesson 5.1. Great job catching 8 bugs! And for doing 2 of the stretch goals!
There is only 1 issue: if you open the console and watch it while you play the game, you'll see an uncaught referenceError appear when you try to enter -2 or 101 into the textbox. This is because you need to move a certain line of code above some other blocks of code. The console message should point you to the location of the error.
Keep up the great work!
-Kira Miller

VadimDmitr committed:
Fixed uncaught referenceError, per Kira's comment, and deleted extra lines of output when entering negative numbers or more than 99

VadimDmitr commented:
Hi Kira,
Thank you!
Fixed uncaught referenceError, per Kira's comment, and deleted an extra line of output when entering negative numbers or more than 99.
